integreat_compass/cms/forms/offers/offer_version_form.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from django import forms
+
+from ...models import OfferVersion
+from ..custom_model_form import CustomModelForm
+
+logger = logging.getLogger(__name__)
+
+
+class OfferVersionForm(CustomModelForm):
+    """
+    Form for creating and editing offer versions
+    """
+
+    prefix = "offer_version"
+
+    class Meta:
+        """
+        This class contains additional meta configuration of the form class, see the :class:`django.forms.ModelForm`
+        for more information.
+        """
+
+        model = OfferVersion
+        fields = ["title", "description", "language", "is_free"]
+        widgets = {"is_free": forms.Select(choices=((True, "Yes"), (False, "No")))}
```

Comment:
```suggestion
        widgets = {"is_free": forms.Select(choices=((True, _("Yes")), (False, _("No"))))}
```
Probably makes sense to translate these? (requires the import)

Reply:
Ah, yes! Sorry, forgot to add.